### PR TITLE
Protobuf cleanup

### PR DESCRIPTION
**Pull request by rkapka**
**What type of PR is this?**

Cleanup

**What does this PR do? Why is it needed?**

- Standard API: remove v2 from all messages that are not request/responses (e.g. BeaconStateV2)
- Prysm API: rename StreamBlocksAltair to StreamBlocks

**Which issues(s) does this PR fix?**

Part of https://github.com/prysmaticlabs/prysm/issues/10322

### CODE REVIEW

**Comment by rkapka** [comment]
Marking as WIP to prevent accidental merging.

File: testing/mock/beacon_validator_client_mock.go [hunk 1]
```diff
@@ -118,26 +118,6 @@ func (mr *MockBeaconNodeValidatorClientMockRecorder) GetBeaconBlock(arg0, arg1 i
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "GetBeaconBlock", reflect.TypeOf((*MockBeaconNodeValidatorClient)(nil).GetBeaconBlock), varargs...)
 }
 
-// GetBlock mocks base method.
-func (m *MockBeaconNodeValidatorClient) GetBlock(arg0 context.Context, arg1 *eth.BlockRequest, arg2 ...grpc.CallOption) (*eth.BeaconBlock, error) {
```
Comment (rkapka) [inline testing/mock/beacon_validator_client_mock.go:122]: This was not updated in https://github.com/prysmaticlabs/prysm/pull/11106

File: testing/mock/beacon_validator_client_mock.go [hunk 2]
```diff
@@ -298,26 +278,6 @@ func (mr *MockBeaconNodeValidatorClientMockRecorder) ProposeBeaconBlock(arg0, ar
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "ProposeBeaconBlock", reflect.TypeOf((*MockBeaconNodeValidatorClient)(nil).ProposeBeaconBlock), varargs...)
 }
 
-// ProposeBlock mocks base method.
-func (m *MockBeaconNodeValidatorClient) ProposeBlock(arg0 context.Context, arg1 *eth.SignedBeaconBlock, arg2 ...grpc.CallOption) (*eth.ProposeResponse, error) {
```
Comment (rkapka) [inline testing/mock/beacon_validator_client_mock.go:302]: This was not updated in https://github.com/prysmaticlabs/prysm/pull/11106

File: testing/mock/beacon_validator_server_mock.go [hunk 3]
```diff
@@ -97,21 +97,6 @@ func (mr *MockBeaconNodeValidatorServerMockRecorder) GetBeaconBlock(arg0, arg1 i
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "GetBeaconBlock", reflect.TypeOf((*MockBeaconNodeValidatorServer)(nil).GetBeaconBlock), arg0, arg1)
 }
 
-// GetBlock mocks base method.
-func (m *MockBeaconNodeValidatorServer) GetBlock(arg0 context.Context, arg1 *eth.BlockRequest) (*eth.BeaconBlock, error) {
```
Comment (rkapka) [inline testing/mock/beacon_validator_server_mock.go:101]: This was not updated in https://github.com/prysmaticlabs/prysm/pull/11106

File: testing/mock/beacon_validator_server_mock.go [hunk 4]
```diff
@@ -232,21 +217,6 @@ func (mr *MockBeaconNodeValidatorServerMockRecorder) ProposeBeaconBlock(arg0, ar
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "ProposeBeaconBlock", reflect.TypeOf((*MockBeaconNodeValidatorServer)(nil).ProposeBeaconBlock), arg0, arg1)
 }
 
-// ProposeBlock mocks base method.
-func (m *MockBeaconNodeValidatorServer) ProposeBlock(arg0 context.Context, arg1 *eth.SignedBeaconBlock) (*eth.ProposeResponse, error) {
```
Comment (rkapka) [inline testing/mock/beacon_validator_server_mock.go:236]: This was not updated in https://github.com/prysmaticlabs/prysm/pull/11106

**Review by rauljordan — DISMISSED** [review]

**Review by rauljordan — APPROVED** [review]

**Pull request merged**